### DISCUSSION TITLE
Get workflow_dispatch value for Workflow Run

### DISCUSSION THREAD

**Discussion by fxs7576** [community/community#27662] [Apps, API and Webhooks]
Hi Github Community,

Is there a way to view the input values of the workflow_display via. Github API?

For some reason, I'm not able to find them using any of these endpoints:
- https://api.github.com/repos/{user}/{repo_name}/actions/workflows
- https://api.github.com/repos/{user}/{repo_name}/actions/workflows/{workflow_id}
- https://api.github.com/repos/{user}/{repo_name}/actions/runs
- https://api.github.com/repos/{user}/{repo_name}/actions/runs/{workflow_id}/jobs

-FS

**Comment by airtower-luna** [comment]
There is no such list mentioned in the documentation, and I can't see it in test output either.

I suppose your best option is to read the `path` attribute of the workflow you want to get the workflow file, and download that (which is possible over the API). Then you can parse the YAML and look at the `on.workflow_dispatch.inputs` element.
